schemas/questionnaire_v1.json

```diff
@@ -137,6 +137,19 @@
     "messages": {
       "$ref": "common_definitions.json#/messages"
     },
+    "hub": {
```

Comment:
We should probably use this in one of the test schemas.

Reply:
Added valid and invalid schemas.